Dockerfile

```diff
@@ -63,19 +67,25 @@ LABEL org.opencontainers.image.revision=${GIT_COMMIT}
 LABEL org.opencontainers.image.created=${BUILD_TIME}
 LABEL org.opencontainers.image.authors=${BUILD_USER}
 
+RUN apt-get update && apt-get install -y locales
+# Generate en_US.UTF-8 locale which is needed to start postgres server.
+# Fix the posgres server issue (invalid value for parameter "lc_messages": "en_US.UTF-8").
+RUN echo "en_US.UTF-8 UTF-8" > /etc/locale.gen && locale-gen
+
 COPY --from=backend /backend-build/bytebase /usr/local/bin/
 
 # Copy utility scripts, we have
 # - Demo script to launch Bytebase in readonly demo mode
 COPY ./scripts /usr/local/bin/
 
 # Create bytebase user for running Postgres database and server.
-RUN addgroup -g 113 -S bytebase && adduser -u 113 -S -G bytebase bytebase
+RUN addgroup --gid 113 --system bytebase && adduser --uid 113 --system bytebase && adduser bytebase bytebase
 
 # Directory to store the data, which can be referenced as the mounting point.
 RUN mkdir -p /var/opt/bytebase
 
 CMD ["--host", "http://localhost", "--port", "80", "--data", "/var/opt/bytebase"]
+EXPOSE 80
```

Comment:
Do we need to explicitly declare `EXPOSE`?

Reply:
Removed.